packages/store/src/types.js

```diff
@@ -224,7 +225,8 @@
  */
 
 /**
- * @template K,V
+ * @template {Key & object} K
```

Comment:
```suggestion
 * @template {Key} K
```
Not weak. Why `& object`? Seems wrong.